src/main/java/seedu/address/logic/command/copy/CopyCommand.java

```diff
@@ -58,8 +58,10 @@ public CommandResult execute(Model model) throws CommandException {
             TravelPlan travelPlan = travelPlanList.get(travelPlanIndex.getZeroBased());
 
             model.copyActivity(activityToCopy, travelPlanIndex);
+            assert travelPlan.getActivityList().hasActivity(activityToCopy) : "Activity was not copied";
```

Comment:
can replace this with:

`assert model.hasTravelPlanObject(activityToCopy, travelPlanIndex.getZeroBased()) : "Activity was not copied";`

This is a new method I recently added to model. Will merge my PR soon. Can pull from upstream to retrieve.